app/models/administrateur.rb

```diff
@@ -74,8 +74,20 @@ def self.reset_password(reset_password_token, password)
   end
 
   def feature_enabled?(feature)
-    ids = Features.send(:"#{feature}_for_admin_ids")
-    ids.present? ? id.in?(ids) : false
+    Flipflop.feature_set.feature(feature)
+    features[feature.to_s]
```

Comment:
Safety check : il me semble que quand on disable une feature, on la `delete` (cf. plus bas), or ici on fait `features[feature.to_s]`, donc (si je ne dis pas de bétises), ça va retourner `nil` si on a désactivé la feature. Donc cette méthode de retourne pas toujours un booléen ?

Reply:
T'es en train de réanimer le vieux drama des predicates en ruby ? :) http://blog.leshill.org/blog/2012/03/25/a-question-of-truth.html

Reply:
J'avais que 3 mois de Ruby dans les pattes à l'époque ;)